package.json

```
@@ -3,55 +3,63 @@
 	"version": "1.0.0",
 	"description": "",
 	"engines": {
-		"node": ">=10.9"
+		"node": ">=12"
 	},
 	"scripts": {
```

```suggestion
	"sideEffects": false,
	"scripts": {
```

Suggested by Webpack: https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

I'm almost certain this is all side-effect free but check me on that